packages/starlight/components/SiteTitle.astro

```diff
@@ -2,28 +2,29 @@
 import { logos } from 'virtual:starlight/user-images';
 import config from 'virtual:starlight/user-config';
 import type { Props } from '../props';
+import { Image } from 'astro:assets';
 const { siteTitle, siteTitleHref } = Astro.props;
 ---
 
 <a href={siteTitleHref} class="site-title sl-flex">
 	{
 		config.logo && logos.dark && (
 			<>
-				<img
+				<Image
 					class:list={{ 'light:sl-hidden': !('src' in config.logo) }}
-					alt={config.logo.alt}
-					src={logos.dark.src}
-					width={logos.dark.width}
+					src={logos.dark}
 					height={logos.dark.height}
+					width={logos.dark.width}
+					alt={config.logo.alt}
```

Comment:
The `<Image>` component loads the image lazily by default, so we should explicitly override that behaviour here given this is high priority content:

```suggestion
					alt={config.logo.alt}
					loading="eager"
```

Reply:
Understood, thanks.